src/styles/sidebar/components/annotation-body.scss

```diff
@@ -0,0 +1,38 @@
+@use '../../variables' as var;
+
+.annotation-body {
+  @include var.font-normal;
+  color: var.$grey-6;
+  // Margin between top of ascent of annotation body and
+  // bottom of ascent of annotation-quote should be ~15px.
+  margin-top: var.$layout-h-margin - 5px;
+  margin-bottom: var.$layout-h-margin;
+  margin-right: 0px;
+  margin-left: 0px;
+}
+
+.annotation-body__text {
+  // Hidden annotations displayed to moderators, where the content is still
+  // present.
+  &.is-hidden.has-content {
+    text-decoration: line-through;
+    filter: grayscale(100%) contrast(65%);
```

Comment:
I see that this is copy-paste, and is fine for now, but might fit in to a color inventory/cleanup at some point.

Reply:
Agreed. This PR was just moving existing stuff around.